app/components/check_answers_summary_list_card_component.html.erb

```diff
@@ -35,8 +35,8 @@
 
             <% if @log.collection_period_open_for_editing? %>
               <% row.with_action(
-                text: question.action_text(log),
-                href: action_href(question, log),
+                text: question.action_text(log, correcting_hard_validation: @correcting_hard_validation),
+                href: @correcting_hard_validation ? correct_validation_action_href(question, log, applicable_questions.map(&:id)) : action_href(question, log),
```

Comment:
Can this be built into action_href at all? (I guess needing to pass in applicable_questions might be a problem)

Reply:
Yes! I updated that on this branch: https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/2483